src/nidigital/unit_tests/test_nidigital.py

```diff
@@ -36,11 +36,68 @@ def teardown_method(self, method):
     # API Tests
 
     # TODO(sbethur): When nidigital driver provides better simulation support (internal bug# 992370),
-    #  this test should be converted to a system test (GitHub issue# 1353).
+    #  this test should be converted to a system test.
+    def test_fetch_history_ram_cycle_information_position_negative(self):
+
+        self.patched_library.niDigital_GetHistoryRAMSampleCount.side_effect = self.side_effects_helper.niDigital_GetHistoryRAMSampleCount
+        self.side_effects_helper['GetHistoryRAMSampleCount']['sampleCount'] = 7
+        with nidigital.Session('') as session:
+            with pytest.raises(ValueError, match='position should be greater than or equal to 0.'):
+                session.sites[1].fetch_history_ram_cycle_information(position=-1, samples_to_read=-1)
+
+    # TODO(sbethur): When nidigital driver provides better simulation support (internal bug# 992370),
+    #  this test should be converted to a system test. (GitHub issue# 1353).
     def test_fetch_history_ram_cycle_information_position_out_of_bound(self):
 
         self.patched_library.niDigital_GetHistoryRAMSampleCount.side_effect = self.side_effects_helper.niDigital_GetHistoryRAMSampleCount
         self.side_effects_helper['GetHistoryRAMSampleCount']['sampleCount'] = 7
         with nidigital.Session('') as session:
             with pytest.raises(ValueError, match='position: Specified value = 8, Maximum value = 6.'):
                 session.sites[1].fetch_history_ram_cycle_information(position=8, samples_to_read=-1)
+
+    # TODO(sbethur): When nidigital driver provides better simulation support (internal bug# 992370),
+    #  this test should be converted to a system test.  
+    def test_fetch_history_ram_cycle_information_is_finite_invalid(self):
```

Comment:
This test can be run on simulated session as system test. You can remove this unit test.

Reply:
Removed